src/sonic-config-engine/sonic-cfggen

```diff
@@ -435,6 +432,7 @@ def main():
         if args.namespace is None:
             configdb = ConfigDBPipeConnector(use_unix_socket_path=True, **db_kwargs)
         else:
+            SonicDBConfig.load_sonic_global_db_config(namespace=args.namespace)
```

Comment:
>SonicDBConfig.load_sonic_global_db_config(namespace=args.namespace) [](http://example.com/codeflow?start=12&length=67)

An extreme case:
if both `args.from_db` and `args.write_to_db` have valid stirng, will the second `load_sonic_global_db_config` throw exception?